clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnectionBuilder.java

```diff
@@ -43,6 +47,11 @@ public HubConnectionBuilder configureLogging(Logger logger) {
         return this;
     }
 
+    public HubConnectionBuilder withHeaders(Map<String, String> headers) {
```

Comment:
Should I add another API to add individual headers?

Reply:
`addHeader(String name, String value)`

Reply:
Yeah, I think that would be good.